src/lib/doc.ts

```diff
@@ -7,12 +7,34 @@
  *
  * The Original Developer is the Initial Developer. The Initial Developer of the Original Code is Atypon Systems LLC.
  *
- * All portions of the code written by Atypon Systems LLC are Copyright (c) 2022 Atypon Systems LLC. All Rights Reserved.
+ * All portions of the code written by Atypon Systems LLC are Copyright (c) 2024 Atypon Systems LLC. All Rights Reserved.
  */
 import { CHANGE_OPERATION } from '@manuscripts/track-changes-plugin'
 import { ManuscriptEditorState, ManuscriptNode } from '@manuscripts/transform'
 
-export function filterUnchangedContent(node: ManuscriptNode) {
+export type PMDoc = any
```

Comment:
Can we improve on that? I think there is a JSONNode type that we use in transform, that probably can be used here instead of any or we can use just some generic JSON type too.

Reply:
The old quarterback code had a `JsonValue` type imported from the `type-fest`  package, but I think it's overkill to have that package just for this type. I can add it back if it's preferred.

Reply:
Alternativally, I can define a general type for JSON and use it.

Reply:
I was thinking that we can use the type from transform - 
[export type JSONNode ](https://github.com/Atypon-OpenSource/manuscripts-transform/blob/master/src/schema/migration/migrate.ts#L23), but anything that at least broadly suggest a document JSON-like structure will do just as well.

Reply:
That type is defined under "migration" so I prefer not to use it in its current state. I would rather not update the transform for now (as it would result in several other components being published), so I can move the type and use it here after this is merged if you don't mind.

Reply:
Ok. Sounds good